plugins/wme-sitebuilder/wme-sitebuilder/Plugin.php

```diff
@@ -133,4 +136,28 @@ public function registerModule( $module ) {
 
 		return $this;
 	}
+
+	/**
+	 * Add the new admin color scheme.
+	 */
+	public function add_admin_color_scheme() {
+		return wp_admin_css_color(
+			'sitebuilder',
+			'WME Sitebuilder',
+			$this->getAssetSource( 'admin/admin-color-scheme.css' ),
+			['#000000', '#ffffff', '#5a00cd' , '#232a2c']
+		);
+	}
+
+	/**
+	 * Set the new admin color scheme to the current user
+	 */
+	public function set_admin_color_scheme() {
+		$user_id = get_current_user_id();
+		$args = [
+			'ID' => $user_id,
+			'admin_color' => 'sitebuilder'
+		];
+		wp_update_user( $args );
```

Comment:
I'm not that familiar with this project, but won't this update the user in the database on every page load?

Reply:
Also, I'm not sure if you want any additional error checking/logging here to make sure this actually worked or not.